deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml

```diff
@@ -27,6 +27,12 @@ data:
         resource-type:
           resource: pods
           version: v1
+    - kind: "k8s-dynamic"
+      name: "k8s/namespaces"
+      config:
+        resource-type:
+          resource: namespaces
+          version: v1
```

Comment:
For other each of the other resources types there is a corresponding RBAC ClusterRole and ClusterRoleBinding which explicitly grants permission to the the agent service account to read those resources.

Do service accounts implicitly have permission to list namespaces?

The RBAC ClusterRole was also omitted in  https://github.com/jetstack/jetstack-secure/pull/507
 * https://github.com/jetstack/jetstack-secure/pull/507